src/TestingUtils/Microsoft.AspNetCore.Testing/test/FlakyAttributeTest.cs

```diff
@@ -0,0 +1,59 @@
+using Microsoft.AspNetCore.Testing.xunit;
+using System;
+using System.Collections.Generic;
+using Xunit;
+
+namespace Microsoft.AspNetCore.Testing.Tests
+{
+    public class FlakyAttributeTest
+    {
+        [Fact]
+        [Flaky("http://example.com")]
+        public void AlwaysFlaky()
+        {
+            throw new Exception("Flakey!");
```

Comment:
Flaky

Reply:
Yep, also I don't want this test to fail for dev builds so I'll have it detect either AzDO or Helix and only fail there.